### PR TITLE
Rl rt fixes

### PR DESCRIPTION
**Pull request by RLuckom**

### CODE REVIEW

File: broad-wb-malkov.json [hunk 1]
```diff
@@ -262,6 +262,22 @@
         }
       }
     },
+    "tos-project": {
```
Comment (RLuckom) [inline broad-wb-malkov.json]: this is fine

File: broad-wb-perf2.json [hunk 2]
```diff
@@ -76,6 +76,10 @@
             "from_path": "secret/dsde/firecloud/perf/common/proxy-ldap",
             "to_path": "common/proxy-ldap"
           },
+          "oauth_client_id":{
```
Comment (RLuckom) [inline broad-wb-perf2.json]: need this secret in each env.

File: profiles/rawls-sa/terraform/rawls/sa.tf [hunk 3]
```diff
@@ -55,5 +62,5 @@ resource "google_service_account_key" "billingprobe_account_key" {
 
 resource "vault_generic_secret" "billingprobe_account_key" {
   path = "${var.vault_path_prefix}/${var.service}/${var.service}-billingprobe-account.json"
-  data_json = "${base64decode(google_service_account_key.app_account_key.private_key)}"
```
Comment (RLuckom) [inline profiles/rawls-sa/terraform/rawls/sa.tf]: this was wrong

File: broad-wb-perf2.json [hunk 4]
```diff
@@ -189,7 +193,7 @@
       "env": {
         "APP_NAME": "rawls",
         "FIRECLOUD_DEVELOP_GIT_BRANCH": "perf-159-rawls",
-        "SERVICE_GIT_SHA_12_CHAR": "perf"
+        "SERVICE_GIT_SHA_12_CHAR": "dev"
```
Comment (Shplorf) [inline broad-wb-perf2.json]: Should we switch em all to `dev`?

Comment (RLuckom) [reply broad-wb-perf2.json]: will do

**Pull request merged**